### PR TITLE
Recorded TV Views and Artwork with some minor fixes

### PR DESCRIPTION
**Pull request by rjsachse**
Added "seeking" string to language #32000

Added addon.xml to repo for future addon updates.

Added pvr images to right list view.

Updated variables for PVR Recorded TV images and extra fanart

Added option in - "Skin Settings" - "Customise/Customize Main Menu" -
"Live TV" - "Background Image - Use Collection Fanarts" (Thanks to
script skin helper for all art work grabbing)

Added Top Menu change views for recorded TV (some more work needed ad
maybe redesign)

Added Media Portal PVR Recording Dialog

Work In Progress...
Please report Bugs...

### CODE REVIEW

**Comment by Angelinas1** [comment]
I merge in to mine file,you have more then 720p folder.
I think just this folder is more than enough....for now.
Try this No.9 ithink its all....hope that I didnt miss something

**Comment by rjsachse** [comment]
Just had to add Australian mpaa flags images

**Comment by rjsachse** [comment]
Sorry forgot to take out the language folder

File: 720p/IncludesTopMenu.xml [hunk 1]
```diff
@@ -1140,7 +1140,7 @@
 				<!-- Global -->
 				<control type="button" id="92149">
 					<include>TopMenuButton</include>
-					<label>DialogSeekBar Art</label>
+					<label>FullScreen Icon</label>
```
Comment (Angelinas1) [inline 720p/IncludesTopMenu.xml:1143]: Change this two label in your git hub version

File: 720p/IncludesTopMenu.xml [hunk 2]
```diff
@@ -1152,7 +1152,7 @@
 				</control>
 				<control type="button" id="92148">
 					<include>TopMenuButton</include>
-					<label>FullScreenInfo Art</label>
+					<label>DialogSeekBar Logo/Clearart</label>
```
Comment (Angelinas1) [inline 720p/IncludesTopMenu.xml:1155]: same

**Pull request merged**